### PR TITLE
[FEAT] Add preference update/set socket event to frontend and backend

### PR DESCRIPTION
**Pull request by dagea**
Added socket event to handle update of preferences from session host. Emits new preferences from backend to frontend clients of the particular session.

### CODE REVIEW

File: backend/src/sockets/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export {joinRoom} from './joinRoom';
 export {disconnect} from './disconnect';
 export {start} from './start';
+export {setPrefences} from '././setPreferences';
```
Comment (afei088) [inline backend/src/sockets/index.js:4]: spelling typo

Comment (afei088) [reply backend/src/sockets/index.js:4]: should be setPreferences :)

Comment (afei088) [reply backend/src/sockets/index.js:4]: also the import path is incorrect, should be './setPreferences' I believe

**Review by afei088 — CHANGES_REQUESTED** [review]
just some typos, everything else looks good

File: backend/src/sockets/index.js (hunk 1 above)
Comment (dagea) [reply backend/src/sockets/index.js]: oops dunno how that second one happened, should be fixed now

**Review by afei088 — APPROVED** [review]

**Review by David-Xia0 — APPROVED** [review]

**Review by Tianrens — APPROVED** [review]

**Pull request merged**